packages/docs/api/api-reference.md

```diff
@@ -1,627 +1,551 @@
 ---
 id: api-reference
-title: API reference
+hide_title: true
```

Comment:
I get that it makes sense to have this file committed right now to make review process simpler but i think when we are done we should gitignore this file and generate it on the fly when building website